src/main/java/com/ryszardpanda/medicalClinic/controller/InstitutionController.java

```diff
@@ -19,4 +17,9 @@ public class InstitutionController {
     public InstitutionDTO addInstitution(@Valid @RequestBody InstitutionDTO institutionDTO){
         return institutionMapper.institutionToInstitutionDTO(institutionService.addInstitution(institutionDTO));
     }
+
+    @PatchMapping("/{institutionId}/assign-doctor/{doctorId}")
```

Comment:
zapytanie do tego endpointu bedzie na sciezce: "localhost:8080/1/assign-doctor/2" np. Czy to okej?